src/magic_folder/test/test_web.py

```diff
@@ -429,13 +444,21 @@ def test_list_folders(self, folders):
                     body_matcher=AfterPreprocessing(
                         loads,
                         Equals({
-                            u"folders": list(
-                                {u"name": name, u"local-path": path_u.path}
-                                for (name, path_u)
-                                in sorted(folders.items())
-                            ),
+                            name: {
+                                u"name": name,
```

Comment:
Reading these two lines, it occurs to me that maybe the name property is now redundant?